File: Tkw.Web.Client.Cordova/www/scripts/app/models/contact-info.ts
module RapApp.Models {
    export interface IContactInfoList extends ng.IScope, BaseModel {
        ContactInfos: Array<IContactInfo>;
        CurrentContactInfo: IContactInfo;
        EditContactInfo: IContactInfo;
        BuildingId: string;
    }
    export interface IContactInfo {
        Id: string;
        BuildingId: string;
        Title: string;
        FirstName: string;
        LastName: string;
        Role: string;
        EmailAddress: string;
        Phone: string;
        MobilePhone: string;
        Address: string;
        SecondAddress: string;
        City: string;
        State: string;
        ZIP: string;
    }
}

File: Tkw.BusinessObjects/BO Domain/ContactInfo.cs
namespace BusinessObjects
{
    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;

    public partial class ContactInfo : BaseDate
    {
        #region Properties
        [Key]
        public virtual string Id { get; set; }

        public virtual string BuildingId { get; set; }
        public virtual string Title { get; set; }
        public virtual string FirstName { get; set; }
        public virtual string LastName { get; set; }
        public virtual string Role { get; set; }
        public virtual string EmailAddress { get; set; }
        public virtual string Phone { get; set; }
        public virtual string MobilePhone { get; set; }
        public virtual string Address { get; set; }
        public virtual string SecondAddress { get; set; }
        public virtual string City { get; set; }
        public virtual string State { get; set; }
        public virtual string Zip { get; set; }
        #endregion

        #region Navigation Properties
        public Building Building { get; set; }
        #endregion
    }
}
